CCVA_Processing.Rmd

```diff
@@ -565,20 +582,36 @@ centroid_monthly <- centroid_all  %>%
             dplyr::group_by(y,m,gcm,unit_climate_zone,cf) %>%
             dplyr::summarize(
               across(all_of(sum_cols),sum),  
-              across(all_of(mean_cols),mean)) %>%
-            dplyr::mutate(ym = make_date(y,m), , .before = gcm) %>%
+              across(all_of(mean_cols),mean),
+              .groups = "keep") %>%
+            dplyr::mutate(ym = make_date(y,m), .before = gcm) %>%
```

Comment:
I've never used the `.before = ` argument before (lol), I'm learning things!! Will definitely be using this in the future.